packages/web/pages/player/[username].tsx

```diff
@@ -345,54 +360,26 @@ export const Grid: React.FC<Props> = ({
           justifyContent="end"
           variant="ghost"
           zIndex={10}
+          isAttached
           h="3rem"
           mb="1rem"
         >
-          {changed && canEdit && !isDefaultLayout && (
-            <MetaButton
-              aria-label="Reset"
-              _hover={{ background: 'purple.600' }}
-              textTransform="uppercase"
-              px={12}
-              letterSpacing="0.1em"
-              size="lg"
-              fontSize="sm"
-              onClick={() => setExitAlertReset(true)}
-              leftIcon={<RepeatClockIcon />}
-              whiteSpace="pre-wrap"
-            >
-              Reset
-            </MetaButton>
-          )}
-          {changed && canEdit && (
+          {changed && editing && (
             <MetaButton
               aria-label="Cancel edit layout"
+              colorScheme="purple"
               _hover={{ background: 'purple.600' }}
               textTransform="uppercase"
               px={12}
               letterSpacing="0.1em"
               size="lg"
               fontSize="sm"
-              onClick={() => setExitAlertCancel(true)}
-              leftIcon={<CloseIcon />}
+              onClick={handleCancel}
```

Comment:
And we want CloseIcon instead of DeleteIcon here as per Dave's design.

Reply:
Changed.